doc/ref_impl/cntt-ri2/chapters/chapter02.md

```diff
@@ -4,21 +4,31 @@
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [2.1 Introduction](#2.1)
+* [2.1 Introduction](#21-introduction)
+  * [2.1.1 Definitions](#211-definitions)
 * [2.2 Reference Architecture Specification](#2.2)
 * [2.3 Reference Implementation Requirements](#2.3)
 
-<a name="2.1"></a>
 ## 2.1 Introduction
 
-> To be completed
+This chapter will use the requirements defined in the Kubernetes Reference
+Architecture and only make additional entries in section [2.3](#2.3) if there
+are additional requirements needed for this Reference Implementation.
+
+## 2.1.1 Definitions
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
 
-<a name="2.2"></a>
 ## 2.2 Reference Architecture Specification
 
-> Pull out the specifications provided in RA2
+| RA2 Section | Reference  | Description | Requirement for Basic Profile | Requirement for Network Intensive Profile| RI2 Traceability |
```

Comment:
- Since the content of the field "RA2 Section" is not the actual section but some "header" (e.g., Kubernetes Nodes) should this be relabeled
- Should Reference be labled "RA2 Reference"?

Reply:
> * Should Reference be labled "RA2 Reference"?

Updated in https://github.com/cntt-n/CNTT/pull/1913/files#diff-066f578f150e408283f6386302730542

Reply:
> * Since the content of the field "RA2 Section" is not the actual section but some "header" (e.g., Kubernetes Nodes) should this be relabeled

Not sure I understand - this refers to the section of the RA2 chapter 4 - Kubernetes Nodes is such a section?